### DISCUSSION TITLE
[Bug]: 文档站的自定义表情出现问题

### DISCUSSION THREAD

**Discussion by w0oo** [walinejs/waline#1344] [Q&A] [answered]
![Screenshot_2022-09-02-18-58-08-94_40deb401b9ffe8e1df2f1cc5ba480b12](https://user-images.githubusercontent.com/105968998/188125675-734807a8-fc3f-4489-813a-eeafabb936de.jpg)

出现了问题，且界面404

@lizheming

**Comment by lizheming** [comment, marked as answer]
https://github.com/walinejs/waline/pull/1345 你的 PR 导致的问题，已回滚

**Reply by w0oo** [reply]
好的
